packages/edit-site/src/components/sidebar-navigation-screen-pages/index.js

```diff
@@ -95,9 +95,11 @@ export default function SidebarNavigationScreenPages() {
 											key={ page.id }
 											withChevron
 										>
-											{ decodeEntities(
-												page.title?.rendered
-											) ?? __( '(no title)' ) }
+											<Truncate numberOfLines={ 1 }>
+												{ decodeEntities(
+													page.title?.rendered
+												) ?? __( 'Untitled' ) }
```

Comment:
I think this should be `( no title )` because that's what we used everywhere in WordPress.